perseo-main/pom.xml

```diff
@@ -78,6 +78,16 @@
             <groupId>com.telefonica.iot</groupId>
             <version>1.5.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20180813</version>
+        </dependency>
+        <dependency>
+            <groupId>org.owasp.encoder</groupId>
+            <artifactId>encoder</artifactId>
+            <version>1.2.2</version>
+		</dependency>
```

Comment:
Wrong indent.

Reply:
Fixed in 05982ba4eefed043b201d8b95eb1c9b6190d2673